_attributes/attributes-openshift-dedicated.adoc

```diff
@@ -20,8 +20,17 @@
 :rhq-short: Red Hat Quay
 :SMProductName: Red Hat OpenShift Service Mesh
 :pipelines-title: Red Hat OpenShift Pipelines
+//logging
+:logging-title: logging subsystem for Red Hat OpenShift
+:logging-title-uc: Logging subsystem for Red Hat OpenShift
+:logging: logging subsystem
```

Comment:
Many entries is the [RH SSG](https://redhat-documentation.github.io/supplementary-style-guide/#batch-jberet) have subsystem names highlighted, so should `logging` follow the same pattern?

Reply:
I'm not sure. We are updating this product name soonish in the docs though AFAIK, so I'll leave it as is for now, since this will be followed up by another Jira (these attributes already exist for OCP).